picamctl/picamctl.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+# akseidel 02/06/2021
+# A script automating the steps to run showmewencam's "camera-ctl".
+# The script will pause for continue or exit if run with an argument.
+
+# edit this next line to be the correct text pattern for your computer.
+portnamepat="tty.usbmodem" #typical for showmewebcam on Appl OS X
+#portnamepat="ttyACM" #probably typical for showmewebcam on Linux
+#portnamepat="tty."
+# The client is an application you want this script to also run. 
+runtheclient="true"
+clientname="Photo Booth"
```

Comment:
MacOs specific, should be based on the platform, a parameter or optional

Reply:
Yes it is MacOS specific. The intent is for educator to make edits, which is one reason these are brought to the script top. The intended users (school setting as webcam attached to microscope camera attachment) would not know what a parameter is. I understand the point. Perhaps there is something that could be done.

Reply:
In combination with the auto discover implementation the script sets these per OS.